code/datums/components/fantasy/suffixes.dm

```diff
@@ -0,0 +1,95 @@
+/datum/fantasy_affix/cosmetic_suffixes
+	placement = AFFIX_SUFFIX
+	alignment = AFFIX_GOOD | AFFIX_EVIL
+
+	var/list/goodSuffixes
+	var/list/badSuffixes
+
+/datum/fantasy_affix/cosmetic_suffixes/New()
+	goodSuffixes = list(
+		"dexterity",
+		"constitution",
+		"intelligence",
+		"wisdom", 
+		"charisma",
+		"the forest",
```

Comment:
creation of that tree simplemob upon command with a 5 minute cooldown (maybe adjusted based on the + of the item?)?